imod_coupler/drivers/dfm_metamod/mf6_wrapper.py

```diff
@@ -1,26 +1,92 @@
-from typing import Optional
+from pathlib import Path
+from typing import Any, Optional, Union
 
 import numpy as np
 from numpy.typing import NDArray
 from xmipy import XmiWrapper
 
 
 class Mf6Wrapper(XmiWrapper):
+    def __init__(
```

Comment:
as discussed offline: some changes in this file should be reverted. We now pass the name of the flow-model and package-keys in the constructor. But that is limiting the general usefulness of this wrapper. These should be passed as function arguments, so that the users of this wrapper have freedom to get data out of any package they want-and not necesarily the ones they gave to the constructor. Either create a new task for that or fix it here.